lib/registries/addon/components/registries-registration-type-facet/component.ts

```diff
@@ -4,32 +4,49 @@ import EmberArray, { A } from '@ember/array';
 import Component from '@ember/component';
 import { task } from 'ember-concurrency';
 import DS from 'ember-data';
+import Features from 'ember-feature-flags/services/features';
+import appConfig from 'ember-get-config';
 import I18N from 'ember-i18n/services/i18n';
+import Toast from 'ember-toastr/services/toast';
+
 import RegistrationSchema from 'ember-osf-web/adapters/registration-schema';
 import requiredAction from 'ember-osf-web/decorators/required-action';
 import Analytics from 'ember-osf-web/services/analytics';
 import defaultTo from 'ember-osf-web/utils/default-to';
-import Toast from 'ember-toastr/services/toast';
-import config from 'registries/config/environment';
+
+import engineConfig from 'registries/config/environment';
 import { SearchOptions } from 'registries/services/search';
 import { ShareTermsFilter } from 'registries/services/share-search';
 import layout from './template';
 
+const {
+    sourcesWhitelist,
+} = engineConfig;
+
+const {
+    featureFlagNames: {
+        enableInactiveSchemas,
+    },
+} = appConfig;
+
 export default class RegistriesRegistrationTypeFacet extends Component.extend({
     fetchRegistrationTypes: task(function *(this: RegistriesRegistrationTypeFacet): any {
         try {
             const metaschemas: RegistrationSchema[] = yield this.store.findAll('registration-schema');
 
-            this.set('registrationTypes', A(
-                metaschemas.mapBy('name').concat([
+            const metaschemaNames = metaschemas.mapBy('name');
+            if (!this.features.isEnabled(enableInactiveSchemas)) {
```

Comment:
Just a drive by, and don't change this for this, but reminder that we are trying not to access flags inside of a component, and instead are passing the flag's value into the component, so we can have the different capabilities on the handbook. It's inconsequential for this PR, but hopefully we can discourage this practice in the future.

Reply:
For reusable components, I completely agree 👍